tests/visual/components/AppSidebar/AppSidebar.spec.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
I' not fond of having cypress tests into the tests folder as this is not how cypress works and not how any other apps in Nextcloud does it. I'd prefer things global and the same across repos :wink: 
https://github.com/nextcloud/text/tree/master/cypress/integration
https://github.com/nextcloud/social/pull/243/files#diff-e011fa10528950fcd9a06a080c3aa384

Reply:
I'd say for components that is a better approach. For E2E testing of apps the test structure is usually based on routes or individual views. Since we only test single components here having them near to the code makes sense imo.

Reply:
@juliushaertl so where would you put them? :)
Alongside components?

Reply:
I put them in the tests folder since I thought of them as tests as well (unit, integration and visual tests). But I am fine with whatever you dedice on.